tools/clang/test/HLSL/out-param-diagnostics.hlsl

```diff
@@ -1,22 +1,22 @@
 // RUN: %clang_cc1 -fsyntax-only -verify %s
 
-void UnusedEmpty(out int Val) {} // expected-warning{{parameter 'Val' is uninitialized when used here}} expected-note{{variable 'Val' is declared here}}
+void UnusedEmpty(out int Val) {} // expected-note {{variable 'Val' is declared here}} expected-warning {{parameter 'Val' is uninitialized when used here}} fxc-pass {{}}
```

Comment:
The only FXC checks here are pretty nonsensical, so if we're going to have this test auto-updated, we should exclude this on FXC since this is testing a feature that doesn't exist in FXC.

Reply:
Changes moved to #5148.